content/en/docs/contribute/intermediate.md

```diff
@@ -513,6 +532,51 @@ most up-to-date version of that branch.
       the branch to your fork, and the pull request should no longer show any
       conflicts.
 
+12. If you have multiple commits on your PR, you have to squash them into a
+    single commit before you PR can be merged. You can check the number of commits
+    on your PR's `Commits` tab or by running `git log` locally. Squashing commits is a form of rebasing.
+
+    ```bash
+    git rebase -i HEAD<number_of_commits>
+    ```  
+
+    The `-i` switch tells git you want to rebase interactively, which enables
+    you to tell git which commits you need to squash into the first one. For
+    example, let's say you have 3 commits on your branch:
+
+    ```bash
+    12345 commit 4 (2 minutes ago)
+    6789d commit 3 (30 minutes ago)
+    456df commit 2 (1 day ago)     
+    ```
+
+    You need to squash your last three commits into the first one.
+
+    ```bash
+    git rebase -i HEAD~3
+    ```
+
+    That command will open up an editor with the following:
+
+    ```bash
+    pick 456df commit 2
+    pick 6789d commit 3
+    pick 12345 commit 4
+    ```
+
+    Change `pick` to `squash` on the commits you want to squash, and make sure
+    the one `pick` commit is at the top of the editor.
+
+    ```bash
+    pick 456df commit 2
+    squash 6789d commit 3
+    squash 12345 commit 4
+    ```
+
+    When you are done, save and close your editor. Then push your squashed
+    commit with `git push -f origin <branch_name>`.
```

Comment:
To me, `git push -f origin <my_new_branch>` is training a bad habit. I recommend *not* including the `-f` flag in a page for this audience. 

I _never_ use `-f` on a push (although I do sometimes use `git push --force-with-lease`, which is safer).
So, I recommend:
```suggestion
    commit with `git push --force-with-lease origin <branch_name>`.
```

Reply:
I agree that `--force-with-lease` is a better option if you are pushing to branch that may have commits from others. Isn't the assumption here that you are pushing to your own fork, which nobody else can push to?  How about listing both options with explanations?   FWIW this is another case where I couldn't push my squashed commit without the `-f` flag.

Reply:
`--force-with-lease` is _always_ a better option, and it's a safe habit to teach.